extensions/wikia/Search/classes/QueryService/Select/Dismax/VideoEmbedTool.php

```diff
@@ -3,38 +3,47 @@
  * Class definition for Wikia\Search\QueryService\Select\Dismax\VideoEmbedTool
  */
 namespace Wikia\Search\QueryService\Select\Dismax;
+
 use Wikia\Search\Utilities;
+
 /**
  * This class is responsible for handling Video Embed Tool search logic.
  */
-class VideoEmbedTool extends Video
-{
+class VideoEmbedTool extends Video {
 	/**
 	 * Here, we overwrite the normal behavior of getFormulatedQuery to search for wiki TOPIC instead of the query
+	 *
 	 * @return string
 	 */
 	protected function getQuery() {
-	  return sprintf( '+(%s) AND ( +(%s)^1000 AND +(%s)^2000 )', $this->getQueryClausesString(), $this->getTopicsAsQuery(), $this->getTransformedQuery() );
+		return sprintf(
+			'+(%s) AND ( +(%s)^1000 AND +(%s)^2000 )',
+			$this->getQueryClausesString(),
+			$this->getTopicsAsQuery(),
+			$this->getTransformedQuery()
+		);
 	}
-	
+
 	/**
 	 * I've noticed that first names or characters are more common in videos, and last names much less common
-	 * This boosts the first token in a query, which often corresponds to a first name. We will probably want to tweak this.
+	 * This boosts the first token in a query, which often corresponds to a first name. We will probably want to tweak
+	 * this.
 	 */
 	protected function getTransformedQuery() {
 		$query = $this->getConfig()->getQuery()->getSolrQuery();
 		$ploded = explode( " ", $query );
 		$first = array_shift( $ploded );
-		if (! empty( $ploded ) ) {
+		if ( !empty( $ploded ) ) {
 			return "{$first}^5 " . implode( " ", $ploded );
 		} else {
 			return $first; // that's all we got. why boost it?
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)